app/forms/instance-create.tsx

```diff
@@ -344,12 +383,15 @@ export function CreateInstanceForm() {
           <Tabs.Trigger value="project" disabled={isSubmitting}>
             Project images
           </Tabs.Trigger>
+          <Tabs.Trigger value="disk" disabled={isSubmitting}>
+            Existing disks
+          </Tabs.Trigger>
         </Tabs.List>
-        {allImages.length === 0 && (
+        {allImages.length === 0 && disks.length === 0 && (
           <Message
             className="mb-8 ml-10 max-w-lg"
             variant="notice"
-            content="Images are required to create a boot disk."
+            content="Images or disks are required to create a boot disk."
```

Comment:
Not that important because very few people should ever see this. But this is confusing because if you're picking a disk you're not creating. The only thing I can think of is

> Images or disks are required to create or attach a boot disk.